src/qrcode/QRCode.tsx

```diff
@@ -39,6 +40,19 @@ export default function QRCodeDisplay({ qrSvgRef }: Props) {
   )
 }
 
+export function StyledQRCodeDisplay({ qrSvgRef }: Props) {
```

Comment:
nit: Would this eventually replace the existing `QRCodeDisplay` or do we want to keep both? If we want to keep both, maybe both should be regular exports or be in separate files (like `StyledQRCodeGen`) instead of `QRCodeDisplay` being a default export?

Reply:
AFAIK, for the time being I think we'll want to keep both -- I'll move the styled QR code wrapper over to its own file.